### PR TITLE
Movie source classes

### PR DESCRIPTION
**Pull request by AlejandroNeira98**
Progress:
- Create Movie class in a separate .rb file.
- Create Source class with an association to the Item class (in a separate .rb file).
- All Movie class properties visible in the diagram should be defined and set up in the constructor method.
- All Source class properties visible in the diagram should be defined and set up in the constructor method.
- Implement methods:
    - add_item method in the Source class
        - should take an instance of the Item class as an input
        - should add the input item to the collection of items
        - should add self as a property of the item object (by using the correct setter from the item object)
    - can_be_archived?() in the Movie class
        - should override the method from the parent class
        - should return true if parent's method returns true OR if silent equals true
        - otherwise, it should return false

### CODE REVIEW

File: movie.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative './item'
+
+class Movie < Item
+  def initialize(date, archived, silent, id: nil)
+    super(date, archived, id: id)
```
Comment (CRClothier) [inline movie.rb:5]: Do you think we need to add attr_accessor for :silent?

File: source.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Source
+  def initialize(name, id: nil)
+    @id = id.nil? ? Time.now.to_i : id
```
Comment (CRClothier) [inline source.rb:3]: And also for :name and :items?

**Review by JuanLPalacios — CHANGES_REQUESTED** [review]
Hi, @AlejandroNeira98  .

Good job so far!
There are some issues that you still need to work on, but you are almost there!

### Highlights:
No linter issues 👍
Clean structured code

### Required Changes: ♻️

- [ ] [required] create getters and setters for the atributes of the classes according to the diagram remember + indicates getter/setter while - indicates only getter


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by CRClothier — CHANGES_REQUESTED** [review]
Hi @AlejandroNeira98,

Great work so far!! 👍 
- Nicely written code. 🥇 
- Professional documentation and commits 🥳

There is just one small thing that may need changing, (though perhaps I have read the diagram wrong)

I think we may need to add attr_accessor for the below comments, what do you think?

**Pull request merged**